cluster-scope/components/resourcequotas/x-small/resourcequotas.yaml

```diff
@@ -0,0 +1,11 @@
+kind: ResourceQuota
+apiVersion: v1
+metadata:
+  name: x-small
```

Comment:
Can you add this new resource quota to the docs as well please?

Reply:
Done: https://github.com/operate-first/support/pull/200